src/zx_diagram.jl

```diff
@@ -67,11 +68,16 @@ function show(io::IO, zxd::ZXDiagram{T, P}) where {T<:Integer, P}
 end
 
 nv(zxd::ZXDiagram) = nv(zxd.mg)
-ne(zxd::ZXDiagram, count_mul::Bool = false) = ne(zxd.mg, count_mul)
-outneighbors(zxd::ZXDiagram, v) = outneighbors(zxd.mg, v)
+ne(zxd::ZXDiagram; count_mul::Bool = false) = ne(zxd.mg, count_mul = count_mul)
+outneighbors(zxd::ZXDiagram, v; count_mul::Bool = false) = outneighbors(zxd.mg, v, count_mul = count_mul)
```

Comment:
add some docstring to this, since I find this name is a bit confusing with others, some docstring would help.